samples/excel/85-preview-apis/data-types-web-image.yaml

```diff
@@ -0,0 +1,206 @@
+order: 2
+id: excel-data-types-web-image
+name: 'Data types: Web images'
+description: This sample shows how to set and get data types using the web image properties.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#insert-image").click(() => tryCatch(insertImage));
+        $("#retrieve-image-info").click(() => tryCatch(retrieveImageInfo));
+        $("#open-image").click(() => tryCatch(openImage));
+        $("#clear-form").click(() => tryCatch(clearForm));
+
+        async function insertImage() {
+          await Excel.run(async (context) => {
+            // This method inserts a web image into the currently selected cell.
+
+            // Retrieve image data from the input fields and then clear the input fields.
+            const imageUrl = $("#url").val() as string;
+            const imageAltText = $("#alt-text").val() as string;
+            clearForm();
+
+            // Load the active cell.
+            const activeCell = context.workbook.getActiveCell();
+            activeCell.load();
+            await context.sync();
+
+            if (!imageUrl) {
+              console.log("Please enter an image URL.");
+              return;
+            }
+
+            // Insert web image into the active cell.
+            activeCell.valuesAsJson = [
+              [
+                {
+                  type: "WebImage",
+                  basicType: "Error",
```

Comment:
Consider adding a /* comment */ explaining why "Error" is used here. Is it a best practice for types that have no basic value equivalent?

Reply:
Good point. Yes, it's a best practice.

Reply:
I misspoke. `basicType` and `basicValue` are readonly properties. They can technically be written, but they have to be written as the values shown here. To simplify and avoid confusion, I will remove these two properties from the Script Lab sample.